### DISCUSSION TITLE
Colourise Tables

### DISCUSSION THREAD

**Discussion by firedaemon-ceo** [GitbookIO/community#146] [Feature Requests]
### What's your feature/idea?

Allow custom colours to be applied to tables. Specifically, table headers, table cells, and text.

### What's your desired outcome?

To be able to colourise tables so they can look like this:

https://kb.firedaemon.com/support/solutions/articles/4000189849

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by CashVo** [comment]
+1 
Also, would be great to be able to define alternating row cell colors ... makes it much easier to scan long tables
